conf/systemd/packetfence-keepalived.service

```diff
@@ -1,8 +1,9 @@
 # Copyright (C) Inverse inc.
 [Unit]
 Description=PacketFence LVS and VRRP High Availability Monitor
-After=packetfence.target packetfence-haproxy-db.service
-Wants=packetfence.target packetfence-iptables.service
+Before=packetfence-haproxy-portal.service
```

Comment:
Shouldn't it be after haproxy-portal is started as in a cluster it would start before the server is ready to offer service

Reply:
It's before because in standalone i won't be able to bind if the ip address is not here.